app/src/main/java/com/duckduckgo/app/onboarding/ui/page/vpn/VpnPermissionPage.kt

```diff
@@ -1,215 +0,0 @@
-import com.duckduckgo.mobile.android.ui.viewbinding.viewBinding
-import com.duckduckgo.mobile.android.vpn.service.TrackerBlockingVpnService
-import com.duckduckgo.mobile.android.vpn.ui.onboarding.DeviceShieldFAQActivity
-import com.duckduckgo.mobile.android.vpn.ui.tracker_activity.AppTPVpnConflictDialog
-import com.duckduckgo.mobile.android.vpn.ui.tracker_activity.AppTPVpnConflictDialog.Listener
-import dagger.android.support.AndroidSupportInjection
-import kotlinx.android.synthetic.main.content_onboarding_default_browser.*
-import kotlinx.android.synthetic.main.include_default_browser_buttons.*
-import kotlinx.coroutines.flow.launchIn
-import kotlinx.coroutines.flow.onEach
-import javax.inject.Inject
-
-@InjectWith(FragmentScope::class)
-class VpnPermissionPage : OnboardingPageFragment(), Listener {
-
-    @Inject
-    lateinit var viewModelFactory: FragmentViewModelFactory
-
-    @Inject
-    lateinit var appBuildConfig: AppBuildConfig
-
-    private val binding: ContentOnboardingVpnPermissionBinding by viewBinding()
-
-    private val viewModel: VpnPagesViewModel by lazy {
-        ViewModelProvider(this, viewModelFactory).get(VpnPagesViewModel::class.java)
-    }
-
-    override fun layoutResource(): Int = R.layout.content_onboarding_vpn_permission
-
-    override fun onAttach(context: Context) {
-        AndroidSupportInjection.inject(this)
-        super.onAttach(context)
-    }
-
-    override fun onActivityResult(
-        requestCode: Int,
-        resultCode: Int,
-        data: Intent?
-    ) {
-        if (requestCode == REQUEST_ASK_VPN_PERMISSION) {
-            viewModel.onAction(VpnPermissionResult(resultCode))
-        }
-        super.onActivityResult(requestCode, resultCode, data)
-    }
-
-    override fun setUserVisibleHint(isVisibleToUser: Boolean) {
-        super.setUserVisibleHint(isVisibleToUser)
-        if (isVisibleToUser) {
-            viewModel.onAction(Action.PermissionPageBecameVisible)
-        }
-    }
-
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
-        observeViewModel()
-        setButtonsBehaviour()
-        binding.onboardingPageText.addClickableLink(
-            "learn_more_link",
-            getText(com.duckduckgo.mobile.android.vpn.R.string.atp_daxOnboardingPermissionMessage)
-        ) {
-            viewModel.onAction(Action.LearnMore)
-        }
-    }
-
-    private fun observeViewModel() {
-        viewModel.commands()
-            .flowWithLifecycle(lifecycle, Lifecycle.State.STARTED)
-            .onEach { processCommand(it) }
-            .launchIn(lifecycleScope)
-    }
-
-    private fun processCommand(command: VpnPagesViewModel.Command) {
-        when (command) {
-            is ShowVpnConflictDialog -> launchVPNConflictDialog(false)
-            is ShowVpnAlwaysOnConflictDialog -> launchVPNConflictDialog(true)
-            is LeaveVpnPermission -> onOnboardingDone()
```

Comment:
As this has been removed we can clean up OnboardingPageFragment. The `fun onOnboardingDone()` declared there is not needed anymore.

Reply:
Will do